library/swipl

```diff
@@ -1,11 +1,11 @@
 Maintainers: Dave Curylo <dave@curylo.org> (@ninjarobot)
 GitRepo: https://github.com/SWI-Prolog/docker-swipl.git
-GitCommit: 1f960ce12029403cd281c659c22213bf70fb4846
+GitCommit: 2fbb0429d19cf02b3332432124b349c62397fa5c
 
 Tags: latest, 7.7.7
 Architectures: amd64, arm32v7
 Directory: 7.7.7/stretch
 
 Tags: stable, 7.6.3
-GitCommit: 1f960ce12029403cd281c659c22213bf70fb4846
+GitCommit: 2fbb0429d19cf02b3332432124b349c62397fa5c
```

Comment:
Btw, you can drop this line entirely in the future if you want -- it's implied by the value that's specified globally. :+1: